bootstrap/yarn.lock

```diff
@@ -1177,6 +1196,13 @@
   dependencies:
     lodash "^4.17.11"
 
+"@gooddata/typings@^2.24.0":
```

Comment:
Please run `yarn-deduplicate --packages "@gooddata/typings"` so that there is only one version here.

Reply:
Fixed. Thanks